app/forms/fields.py

```diff
@@ -28,7 +28,7 @@
 
 
 def get_field(
-    answer, label, error_messages, answer_store, metadata, disable_validation=False
+    answer, label, error_messages, answer_store, metadata, location, disable_validation=False
```

Comment:
If this is defaulted as a kwarg to None, it would save adding it to each of the calling methods.